### PR TITLE
Fix more action things

### PR DESCRIPTION
**Pull request by Orzelius**
Fix some logic and some tests, mostly related to actions and module resolution

Please see individual commits for more detail.

### CODE REVIEW

File: core/src/graph/actions.ts [hunk 1]
```diff
@@ -276,7 +276,7 @@ export async function executeAction<T extends Action>({
 
   const results = await garden.processTasks({ tasks: [task], log, throwOnError: true })
 
-  return <Executed<T>>(<unknown>results.results.getResult(task)!.result!.outputs.executedAction)
+  return <Executed<T>>(<unknown>results.results.getResult(task)!.result!.executedAction)
```
Comment (vvagaytsev) [inline core/src/graph/actions.ts:279]: I'm not that deep into actions router yet, but should the same fi be applied to `resolveAction(s)` functions in https://github.com/garden-io/garden/blob/graph-v2/core/src/graph/actions.ts#L209 and https://github.com/garden-io/garden/blob/graph-v2/core/src/graph/actions.ts#L248?

File: core/test/unit/src/actions.ts [hunk 2]
```diff
@@ -532,45 +489,35 @@ describe("ActionRouter", () => {
...
-        })
+
+        expect(result.outputs).to.eql({})
+        expect(result.detail?.log).to.eql("bla bla")
+        expect(result.state).to.eql("ready")
       })
     })
 
     it("should emit a testStatus event", async () => {
+      const all = graph.getActions()
```
Comment (vvagaytsev) [inline core/test/unit/src/actions.ts:507]: Unused variable declaration

**Review by vvagaytsev — CHANGES_REQUESTED** [review]
Looks good, but I left a couple of comments. Please check.

File: core/test/unit/src/actions.ts (hunk 2 above)
Comment (Orzelius) [reply core/test/unit/src/actions.ts]: good catch, had this left from debugging. [addea3b](https://github.com/garden-io/garden/pull/3264/commits/addea3bace2ad5bd63a7194112487af8725dca9a)

**Review by vvagaytsev — APPROVED** [review]

**Pull request merged**